examples/src/main/kotlin/example/AxiosSearch.kt

```diff
@@ -0,0 +1,87 @@
+@file:Suppress("UnsafeCastFromDynamic")
+
+package example
+
+/**
+ * An example to show how to leverage axios lib to fetch remote data by Scott_Huang@qq.com (Zhiliang.Huang@gmail.com)
+ * (Btw, Vue.js formally leverage axios for ajax related works, so it should be good if React.js also use it)
```

Comment:
React is a rendering library not a framework, so working with network is out of its scope

Reply:
Yeah, agree.
But better add axios example into kotlin-react for people to exchange remote data :)